### DISCUSSION TITLE
Name of object type returned by `simulate_chains()`

### DISCUSSION THREAD

**Discussion by sbfnk** [epiverse-trace/epichains#185] [General]
Should this change from `epichains_tree`, given that we're removing `simulate_tree()` as per https://github.com/epiverse-trace/epichains/issues/181?

Suggestion: `epichain`??

**Comment by jamesmbaazam** [comment]
Yes, you're right. In general, we're also trying to stick with "chains" and not mix it with the use of "trees". 

In the past, I've used `epichains_tree` in anticipation of this package growing to simulate different types of branching processes, if for example, we decide to simulate multi-type processes like the one suggested in #15. Else, `<epichains>` is also fine for now.

Thoughts?

**Comment by jamesmbaazam** [comment]
This has now been resolved in #218 and will be closed. Thanks for raising.
